numpy/lib/tests/test_io.py

```diff
@@ -2539,6 +2540,7 @@ def test_save_load_memmap(self):
                 break_cycles()
                 break_cycles()
 
+    @pytest.mark.skipif(IS_WASM, reason="memmap doesn't work correctly")
```

Comment:
Similarly, maybe xfail this. Someone could try to report it to emscripten and see if it's fixable.

Reply:
We should double check but I think they know about the problem with memmap. I believe it is fixable but nobody has gotten to it.